be/src/types/hll.h

```diff
@@ -177,4 +178,77 @@ class HyperLogLog {
     }
 };
 
+const static datasketches::target_hll_type HLL_TGT_TYPE = datasketches::HLL_6;
+
+class DataSketchesHll {
```

Comment:
can `DataSketchesHll ` only be used in the query SQL? cannot be used as in the create ddl? am I right?

Reply:
Yes, you are right.